tests/__mocks__/vdom.js

```diff
@@ -101,6 +101,76 @@ export const tree = {
     node: null,
 }
 
+export const treeWithStyledComponents = {
+    name: undefined,
+    children: [
+        {
+            name: 'TestWrapper',
+            props: { myProps: 'test prop' },
+            state: { initialized: true },
+            children: [
+                {
+                    name: {
+                        componentStyle: {
+                            rules: [],
+                            isStatic: false,
+                            componentId: 'styled__Button-sc-1fuu6r1-1',
+                        },
+                        displayName: 'styled__Button',
+                        styledComponentId: 'styled__Button-sc-1fuu6r1-1',
```

Comment:
I see that you add the `displayName` property in the util function, but where does the `styledComponentId` and `componentStyle` properties come from?

Reply:
This properties come from styled components but do not need for the test actually